DOCSP-33948: Atomic typos and cleanup (#829)
mongodb/docs-node: source/fundamentals/crud/compound-operations.txt

Overview
--------

> Most database requests either read data out of a database or write data into

Reviewer: S:

```suggestion
Most database requests either read data from a database or write data into
```
